integration_tests/features/support/world.js

```diff
@@ -106,11 +106,11 @@ class CustomWorld {
     this.walletPubkeys[name] = walletInfo.public_key;
   }
 
-  async createAndAddFFIWallet(name, seed_words) {
+  async createAndAddFFIWallet(name, seed_words = null, passphrase = null) {
```

Comment:
technically `seed_words` and `passphrase` would be `undefined` and not `null` without these default values - but since they are both falsy @philipr-za you can leave these defaults out

Reply:
The difference between null and undefined is important for how they get used.  It needs to be programmatically empty rather than a reference is non-existent (very subtle difference). 

Have removed them here and done the check deeper in the wrapper though.